proposals/Exceptions.md

```diff
@@ -240,19 +234,21 @@ If the `kind` is `Exception`:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| type  | `varuint32` | exception index to table in the *exception type section* |
+| tag   | varuint32 | index into exception section |
```

Comment:
Without the tag, how does the catch instruction know to catch an imported exception? We need some way to refer to the imported exception in the catch!